package/gluon-config-mode-geo-location/src/fr.po.in

```diff
@@ -0,0 +1,51 @@
+msgid ""
+msgstr ""
+#ifdef WITHMAP
+"Project-Id-Version: gluon-config-mode-geo-location-with-map\n"
+#else
+"Project-Id-Version: gluon-config-mode-geo-location\n"
+#endif
+"PO-Revision-Date: 2017-08-22 12:14+0100\n"
+"Last-Translator: Jan-Tarek Butt <tarek@ring0.de>\n"
+"Language-Team: French\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"If you want the location of your node to be displayed on the map, you can "
```

Comment:
"can" -> "may"  (same as above)

Reply:
I think you just mean the second "you can"

Reply:
"If you want the location of your node to be displayed on the map, you may "

Reply:
I think in the first sentence should be you can and in the second you may to have a stronger suggestion on the map.